tools/generator/program.go

```diff
@@ -42,11 +42,16 @@ import (
 	"github.com/marstr/collection"
 )
 
+const (
+	defaultGenVer = "@microsoft.azure/autorest.go@v2.0.41"
```

Comment:
I'd say if it's unspecified, we shouldn't provide the "use" flag at all. That way we use whatever the most recent release of autorest.go has been published.

Reply:
Do we emit the version used anywhere so there's a record?  I'd hate to leave it up to fate so we can't recreate how an SDK was generated.

Reply:
@jhendrixMSFT and I chatted offline, we're going to pull out my suggestion for now, and fix it when submit a seperate PR to brand the version of Autorest.Go that was used. Once that happens, we'll take a look at what the default behavior here should be.